beta/src/pages/community/meet-the-team.md

```diff
@@ -7,85 +7,85 @@ React development is led by a small dedicated team working full time at Facebook
 
 [@gaearon on GitHub](https://github.com/gaearon) &middot; [@dan_abramov on Twitter](https://twitter.com/dan_abramov)
 
 Dan got into programming after he accidentally discovered Visual Basic inside Microsoft PowerPoint. He has found his true calling in turning [Sebastian](#sebastian-markbåge)'s tweets into long-form blog posts. Dan occasionally wins at Fortnite by hiding in a bush until the game ends.
 
-### Luna Ruan {#luna-ruan}
+### Luna Ruan {/*luna-ruan*/}
 
 ![Luna](../images/team/lunaruan.jpg)
 
 [@lunaruan on GitHub](https://github.com/lunaruan) &middot; [@lunaruan on Twitter](https://twitter.com/lunaruan)
 
 Luna learned programming because she thought it meant creating video games. Instead, she ended up working on the Pinterest web app, and now on React itself. Luna doesn't want to make video games anymore, but she plans to do creative writing if she ever gets bored.
 
-### Marco Salazar {#marco-salazar}
+### Marco Salazar {/*marco-salazar*/}
 
 ![Marco](../images/team/salazarm.jpeg)
 
 [@salazarm on GitHub](https://github.com/salazarm) &middot; [@BkOptimism on Twitter](https://twitter.com/BkOptimism)
 
 Marco's first programming language was Assembly because he could use it to hack video games. Now online games are much more secure so he settles for playing fairly (mostly). In his spare time he plays games on his treadmill desk and makes art that he never finishes. Hopefully his PRs don't have the same fate.
 
-### Rachel Nabors {#rachel-nabors}
+### Rachel Nabors {/*rachel-nabors*/}
 
 ![Rachel](../images/team/rnabors.jpg)
 
 [@rachelnabors on GitHub](https://github.com/rachelnabors) &middot; [@rachelnabors on Twitter](https://twitter.com/rachelnabors)
 
 Rachel wrote a [book about UI animation](https://abookapart.com/products/animation-at-work) once and worked with MDN and the W3C on the web animations API. Now she is busy with education materials and community engineering on the React team. Secretly, she is an award-winning cartoonist for teenage girls. Catch her making fancy tea with lukewarm water in the microkitchen.
 
-### Rick Hanlon {#rick-hanlon}
+### Rick Hanlon {/*rick-hanlon*/}
 
 ![Ricky](../images/team/rickhanlonii.jpg)
 
 [@rickhanlonii on GitHub](https://github.com/rickhanlonii) &middot; [@rickhanlonii on Twitter](https://twitter.com/rickhanlonii)
 
 Ricky majored in theoretical math and somehow found himself on the React Native team for a couple years before joining the React team. When he's not programming you can find him snowboarding, biking, climbing, golfing, or closing GitHub issues that do not match the issue template.
 
-### Sebastian Markbåge {#sebastian-markbåge}
+### Sebastian Markbåge {#sebastian-markbåge} {/*sebastian-markbåge-sebastian-markbåge*/} {/*sebastian-markbåge-sebastian-markbåge-sebastian-markbåge-sebastian-markbåge*/} {/*sebastian-markbåge-sebastian-markbåge-sebastian-markbåge-sebastian-markbåge-sebastian-markbåge-sebastian-markbåge-sebastian-markbåge-sebastian-markbåge*/}
```

Comment:
Is something wrong here ? since we have special character ?

Reply:
nice catch! Will investigate!

Reply:
👍🏻

Reply:
fixed! indeed, every generate was adding another “ID”

Reply:
Nice work! 😄